src/index.html

```diff
@@ -45,6 +47,65 @@
       </div>
     </div>
 
-  </div>
+
+    <div class = "titleframe">
+      <button class="button button__input">Title:</button>
+      <input class ="title__input">
+    </div>
+
+    <div class ="bodyframe">
+      <button class="button button__input">Body:</button>
+      <textarea class="body__input"></textarea>
+    </div>
+
+    <div class="positionframe">
+
+      <button class="button button__input">Position:</button>
+      <select class="position__input">
+        <option>Top Left</option>
+        <option>Top Right</option>
+        <option>Bottom Left</option>
+        <option>Bottom Right</option>
+      </select>
+
+      <div class="storypointframe">
+        <button class="button__storypoint">Storypoint<h2 class="storypoint__number">1</h2></button>
+        <button class="button__storypoint">Storypoint<h2 class="storypoint__number">2</h2></button>
+        <button class="button__storypoint--add">Add Storypoint<h2 class="storypoint__number storypoint__number--add">+</h2></button>
+      </div>
+
+    </div>
+
+    <br><br>
+
+      <div class="dimensions__field">
+
+        <h2>Fields</h2>
+
```

Comment:
This is a nice example of when something consists of multiple components. First we have a list of things (could be anything). Within that list we have some draggable items.
```html
<div class="list">
  <div class="list__item draggable">...</div>
</div>
```
Each item also has two modifiers. When it's being dragged and a placeholder for the item being dragged. To preview how it looks, make a list like this:
```html
<div class="list">
  <div class="list__item draggable">...</div><!-- Normal -->
  <div class="list__item draggable--dragged">...</div><!-- When it's being dragged -->
  <div class="list__item draggable--placeholder">...</div><!-- Placeholder for the element being dragged -->
</div>
```